src/main/common/maths.h

```diff
@@ -21,16 +21,10 @@
 #define sq(x) ((x)*(x))
 #endif
 
-#ifdef M_PI
-// M_PI should be float, but previous definition may be double
-# undef M_PI
-#endif
-#define M_PI       3.14159265358979323846f
-
-#define RADX10 (M_PI / 1800.0f)                  // 0.001745329252f
-#define RAD    (M_PI / 180.0f)
+// Use floating point M_PI instead explicitly.
+#define M_PIf       3.14159265358979323846f
 
-#define DEG2RAD(degrees) (degrees * RAD)
+#define RAD    (M_PIf / 180.0f)
```

Comment:
Personally I'd prefer the cast, but testing proves that it doesn't make a
difference what way it's done! (so there's no reason to tweak it)